consul/api.go

```diff
@@ -43,3 +64,37 @@ func (c *Coordinator) WithStateListener(
 	result.listener = listener
 	return &result
 }
+
+// PutPStoreConfig stores a new scotty config file
+func (c *Coordinator) PutPStoreConfig(value string) error {
+	return c.coord.kernel.Put(kConfigFileKey, value)
```

Comment:
I'm surprised to see "kernel" here. Is this the OS kernel? I see other references to it as well. If it's not the OS kernel, this seems confusing.